apps/ios/Shared/Views/NewChat/NewChatView.swift

```diff
@@ -487,6 +502,12 @@ private struct ActiveProfilePicker: View {
                     incognitoProfileImage()
                     Text("Incognito")
                         .foregroundColor(theme.colors.onBackground)
+                    Image(systemName: "info.circle")
+                        .foregroundColor(theme.colors.primary)
+                        .font(.system(size: 14))
+                        .onTapGesture {
+                            showIncognitoSheet = true
+                        }
```

Comment:
add Spacer? should look better with info in the end. I know there's a checkmark on selection - maybe it should replace info, or be placed left from it

Reply:
not better. Let's keep where it was before please

Reply:
The reason why it is a bad idea to put info in the end is because if you are right handed, as the majority of people are, you are tapping the row where you see the tick. Now you have info there, so you can't tap this place...